packages/toast/tests/unit/ui-toast.spec.ts

```diff
@@ -1,12 +1,194 @@
-import { shallowMount } from '@vue/test-utils'
-import { UiToastContainer } from '../../src'
+import { mount, VueWrapper } from '@vue/test-utils'
+import { ToastMessageQueueService } from '../../src/toast-message-queue.service'
+import { UiToastContainer, UiToastService } from '../../src'
+import { bsUiToastThemeConfigDefaults } from '../../src/defaults/bs-ui-toast-theme.config'
+import { Position } from '@unicodernsui/core'
 
 describe('ui-toast-container.vue', () => {
-  it('renders when props are set', () => {
-    const wrapper = shallowMount(UiToastContainer, {
-      props: {},
-    })
+  const theme = bsUiToastThemeConfigDefaults
+  let wrapper: VueWrapper<any>
+  let service: UiToastService | undefined
+
+  beforeEach(async () => {
+    wrapper = mount(UiToastContainer)
+
+    service = ToastMessageQueueService.singletonInstance
+    service.clear()
+  })
+
+  it('should render when props are set', () => {
+    const wrapper = mount(UiToastContainer)
```

Comment:
I wonder if this wrapper could be the same declared in line 13

Reply:
solved, thanks